syntax="proto3";
option optimize_for = CODE_SIZE;

message Cfg_SkillFAM
{
    message Row 
	{
		uint32 idskill = 1;
        uint32 dis = 2;				//0=near 1=mid 2=far
		uint32 self_hp = 3;			//0=low 1=ok 2=lots
		uint32 self_mp = 4;			//0=low 1=ok 2=lots
        uint32 target_hp = 5;		//0=low 1=ok 2=lots
		uint32 oper_type = 6;		//0=AND 1=OR
		uint32 like = 7;			//0=unlike 1=like 2=verylike

		
    }

    repeated Row rows = 1;
}